client/state/action-types.ts

```diff
@@ -659,6 +659,10 @@ export const PLUGIN_UPLOAD_ERROR = 'PLUGIN_UPLOAD_ERROR';
 export const PLUGIN_UPLOAD_PROGRESS = 'PLUGIN_UPLOAD_PROGRESS';
 export const PLUGINS_MARKETPLACE_PRIMARY_DOMAIN_CANDIDATE_UPDATE =
 	'PLUGINS_MARKETPLACE_PRIMARY_DOMAIN_CANDIDATE_UPDATE';
+export const PLUGINS_MARKETPLACE_SET_PLUGIN_TO_BE_INSTALLED =
+	'PLUGINS_MARKETPLACE_SET_PLUGIN_TO_BE_INSTALLED';
+export const PLUGINS_MARKETPLACE_SET_IS_PLUGIN_INSTALLED_DURING_PURCHASE =
```

Comment:
Nit: I know these are meant to be self-explicable but I wonder if we could shorten them a bit somehow 😉

Reply:
I agree with this, did some shortening of this while retaining that "MARKETPLACE" prefix 👍 
Fixed by c8f063aecb8458ad5440d8013d0436080258a257

Reply:
Agreed! Fixed by 3f32da1c224b220d2cc936008ea83940bcd5fbd9